index.html

```diff
@@ -2663,12 +2663,14 @@ <h2 id="docconformance">
             </td>
             <td>
               <p>
-                If a descendant of a `table` element, <code>role=<a href="#index-aria-columnheader">columnheader</a></code> or
-                <a href="#index-aria-rowheader">`rowheader`</a> or <a href="#index-aria-rowheader">`cell`</a> (if not a header).
+                If the ancestor `table` element has `role=table`: <code>role=<a href="#index-aria-columnheader">columnheader</a></code>,
+                <a href="#index-aria-rowheader">`rowheader`</a> or <a href="#index-aria-rowheader">`cell`</a> according to
+                <a data-cite="html-aam-1.0#html-element-role-mappings">HTML AAM</a>.
              </p>
               <p>
-                If a descendant of a `table` element with `role=grid` or `treegrid`, <code>role=<a href="#index-aria-columnheader">columnheader</a></code> or
-                <a href="#index-aria-rowheader">`rowheader`</a> or <a href="#index-aria-rowheader">`gridcell`</a> (if not a header).
+                If the ancestor `table` element has `role=grid` or `treegrid`: <code>role=<a href="#index-aria-columnheader">columnheader</a></code>,
```

Comment:
```suggestion
                If the ancestor `table` element is exposed as a `role=grid` or `treegrid`: <code>role=<a href="#index-aria-columnheader">columnheader</a></code>,
```

again, changed this for consistency, but could well see this instance staying "has", since a table would not be exposed as these roles without the explicit declaration.  I could go either way here.